tutor/index.coffee

```diff
@@ -31,6 +31,9 @@ window._STORES =
   NOTIFICATIONS: require './src/flux/notifications'
   TOC: require './src/flux/toc'
 
+# In dev builds this enables hot-reloading,
+# in production it simply renders the root app
+{AppContainer} = require 'react-hot-loader'
```

Comment:
hmm...also... [comment about not needing `AppContainer` anymore maybe?](https://github.com/gaearon/redux-devtools/commit/64f58b7010a1b2a71ad16716eb37ac1031f93915#commitcomment-19376847)